jobfunnel/config/parser.py

```diff
@@ -151,6 +152,13 @@ def parse_cli():
                         help='The maximum number of days old a job can be.'
                         '(i.e pass 30 to filter out jobs older than a month)')
 
+    parser.add_argument('-g',
```

Comment:
I think this setting should not be passed by CLI since it is a beta feature.

We should default to GlassDoorStatic or Dynamic (whichever one works best right now) in the default settings, and allow configuration in YAML.